src/artemis/system_tests/test_main_system.py

```diff
@@ -88,6 +93,12 @@ def check_status_in_response(response_object, expected_result: Status):
     assert response_json["status"] == expected_result.value
 
 
+def test_experiment_registry_and_parameter_model():
+    assert validate_parameter_model_against_registry
+    assert validate_registry_against_parameter_model
+    assert parameter_model_and_plan_registry_consistent
```

Comment:
Should: Because these aren't actually called you're just asserting the functions exist. If you take the above suggestion this test isn't required anyway